datagrid_gtk3/db/sqlite.py

```diff
@@ -269,6 +282,14 @@ def update(self, params, ids=None):
                     cursor.execute(sql)
                 conn.commit()
 
+        for (db_file, table_name, id_), db in self.__class__._DBS.items():
+            if db is self:
+                continue
+            if (db_file, table_name) != (self.db_file, self.table.name):
+                continue
+
+            db.emit('rows-changed', params, ids)
```

Comment:
The signal for the current object is not emitted. The reason for that is because of the use case here: The user selected a row on datagrid and we want other places using the same view (possibly another datagrid connected to the same db/table) to reflect those changes instead of displaying outdated data.

I added a comment on the code to try to clarify that. Hope it is explanatory enough (please tell me if it isn't =P)